pkg/ibgu/ibgu_test.go

```diff
@@ -432,3 +440,200 @@ func generateIbgu() *v1alpha1.ImageBasedGroupUpgrade {
 		Spec: v1alpha1.ImageBasedGroupUpgradeSpec{},
 	}
 }
+
+//nolint:funlen
+func TestPullIgu(t *testing.T) {
+	generateibgu := func(name, namespace string) *v1alpha1.ImageBasedGroupUpgrade {
+		return &v1alpha1.ImageBasedGroupUpgrade{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+			},
+			Spec: v1alpha1.ImageBasedGroupUpgradeSpec{},
+		}
+	}
+
+	testCases := []struct {
+		ibguName            string
+		ibguNamespace       string
+		expectedError       bool
+		addToRuntimeObjects bool
+		expectedErrorText   string
+		client              bool
+	}{
+		{
+			ibguName:            "test1",
+			ibguNamespace:       "test-namespace",
```

Comment:
can use testIbguName and testIbguNamespace here

Reply:
Fixed- thanks.